addons/sale/tests/test_sale_order.py

```diff
@@ -25,6 +30,22 @@ def setUpClass(cls):
         cls.setUpClassicProducts()
         cls.setUpAccountJournal()
 
+        # Create a pricelist with/without discount policy
+        cls.pricelist_with_disct = cls.Pricelist.create({
```

Comment:
use complete name: pricelist_with_disct --> pricelist_with_discount. Saving 3 letters is not very usefull.